dashboard/app/views/home/_homepage.html.haml

```diff
@@ -95,5 +95,6 @@
       cleverLinkFlag: "#{session['clever_link_flag']}",
       userIDToMerge: "#{session['clever_takeover_id']}",
       mergeAuthToken: "#{session['clever_takeover_token']}",
+      forceConnect: "#{session['force_clever_takeover']}",
```

Comment:
Does this need to be a string, or can we convert it to a boolean here right where the value gets stored in JavaScript, instead of inside the React component?

Reply:
Good call, will make this change.

Reply:
Actually, I tried to fix this, and it seems to have seriously broken some stuff, so I'm going to put this off - ultimately this code will be going away, so it'll disappear at that point.